src/zenml/cli/utils.py

```diff
@@ -115,14 +115,14 @@ def pretty_print(obj: Any) -> None:
     click.echo(str(obj))
 
 
-def print_table(obj: List[Dict[str, str]]) -> None:
+def print_table(obj: List[Dict[str, Any]]) -> None:
```

Comment:
The tabulate docs say:

```
he following tabular data types are supported:

list of lists or another iterable of iterables
list or another iterable of dicts (keys as columns)
dict of iterables (keys as columns)
two-dimensional NumPy array
NumPy record arrays (names as columns)
pandas.DataFrame
```
Then maybe we can change this to `List[Dict[str, Iterable]])`

Reply:
Here we are in the case of
-> list or another iterable of dicts (keys as columns)
On the level of the dict values we just expect anything that has a string representation I think.

Reply:
in our case we get str or objects as dict values, I could change this to `List[Dict[str, Any[str, object]]` if you prefer